addons/im_livechat/demo/im_livechat_channel/im_livechat_session_1.xml

```diff
@@ -70,6 +70,7 @@
             <field name="res_id" ref="im_livechat.livechat_channel_session_1"/>
             <field name="author_id" ref="base.public_partner"/>
```

Comment:
Note that if you're fixing demo data, livechat no longer uses public partner but guest instead, so all of those author_id are incorrect

Reply:
It looks like it was fixed on master, maybe not necessary to do it here again. I guess we can just approve the type change.